kolibri/core/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
```

Comment:
Yup, only on startup, but these PRAGMAs will run for the lifetime of the connection -- which is until we shut the server down. That's the effect we want right? For these optimizations to run while the server is running? Or maybe I'm missing something...